js/components/DesignCardComponent.js

```diff
@@ -123,7 +123,7 @@ class DesignCardComponent extends Component {
         if(design.attachments != null && design.attachments.length > 0) {
             downloads = `Attachments: `
             design.attachments.forEach((attachment) => {
-                downloads += `<a href="${attachment.url}" target="_blank" download> ${attachment.name}</a>`;
```

Comment:
Super minor, but could we add `rel="noopener noreferrer"` to the links? Recently found out about this vulnerability 😬 

https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/

Reply:
Of course, added this change, landing it